lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -577,15 +577,18 @@ def set_zsort(self, zsort):
 
     def get_vector(self, segments3d):
         """Optimize points for projection."""
```

Comment:
It would be good to document the parameter in the docstring. This would include also the information of the comment in the following line.